Dockerfile

```diff
@@ -4,6 +4,9 @@ LABEL "repository"="https://github.com/m0nhawk/conda-package-publish-action"
 LABEL "maintainer"="Andrew Prokhorenkov <andrew.prokhorenkov@gmail.com>"
 
 RUN conda install -y anaconda-client conda-build
+RUN conda install -c anaconda make
```

Comment:
do you need `make` explicitly? Is it not in `build-essential`?

Reply:
I'll try it without make

Reply:
You're correct, it works